packages/trader/src/AppV2/Components/Guide/Description/ContractDescription/vanillas-trade-description.tsx

```diff
@@ -0,0 +1,108 @@
+import React from 'react';
+import { Localize } from '@deriv/translations';
+import { CONTRACT_LIST, TERM, getContractDescription } from 'AppV2/Utils/trade-types-utils';
+
+const VanillasTradeDescription = ({ onTermClick }: { onTermClick: (term: string) => void }) => {
+    const content = [
+        {
+            type: 'general',
+            text: (
+                <Localize i18n_default_text='Vanilla options allow you to predict an upward (bullish) or downward (bearish) direction of the underlying asset by purchasing a “Call” or a “Put”.' />
+            ),
+        },
+        {
+            type: 'heading',
+            text: <Localize i18n_default_text='Call' />,
+        },
+        {
+            type: 'paragraph',
+            text: (
+                <Localize
+                    i18n_default_text='If you select “<0>Call</0>”, you’ll earn a <1>payout</1> if the <2>final price</2> is above the <3>strike price</3> at <4>expiry</4>. Otherwise, you won’t receive a payout.'
+                    components={[
+                        <span className='description__content--bold' key={0} />,
+                        <button
+                            className='description__content--definition'
+                            key={1}
+                            onClick={() => onTermClick(TERM.PAYOUT)}
+                        />,
+                        <button
+                            className='description__content--definition'
+                            key={2}
+                            onClick={() => onTermClick(TERM.FINAL_PRICE)}
+                        />,
+                        <button
+                            className='description__content--definition'
+                            key={3}
+                            onClick={() => onTermClick(TERM.STRIKE_PRICE)}
+                        />,
+                        <button
+                            className='description__content--definition'
+                            key={4}
+                            onClick={() => onTermClick(TERM.EXPIRY)}
+                        />,
+                    ]}
+                />
+            ),
+        },
+        {
+            type: 'video',
+            text: `${CONTRACT_LIST.VANILLAS}_call`,
```

Comment:
Same here, I assume you're just importing the word `vanilla` here? What is the point?

Reply:
Yep) Sometimes I'm going crazy with constants, sorry)))
Will fix here and it the similar places